src/debug.c

```diff
@@ -871,6 +871,10 @@ NULL
     {
         stringmatchlen_fuzz_test();
         addReplyStatus(c,"Apparently Redis did not crash: test passed");
+    } else if (!strcasecmp(c->argv[1]->ptr,"set-disable-deny-scripts") && c->argc == 3)
+    {
+        server.lua_disable_deny_script = atoi(c->argv[2]->ptr);;
+        addReplyStatus(c,"OK, disabling deny-scripts is dangerous and should not be done on production");
```

Comment:
Any reason not to just reply `OK` here? Many `DEBUG` commands are dangerous as it is and returning such status replies is not such a common practice.